app/components/new-signup/checkbox.hbs

```diff
@@ -0,0 +1,31 @@
+<div class='user-details'>
+  <h2 class='user-details__heading' data-test-signup-form-label>
+    {{this.label}}
+  </h2>
+
+  <div class='role-details__field'>
+    {{#each this.checkboxData as |data|}}
+      <label class='checkbox-label'>
+        <input

```

Comment:
Can we use Ember <Input /> component by default it will give use adde benefits like a11y